src/api/decommission-service/index.js

```diff
@@ -0,0 +1,18 @@
+import { decommissionServiceController } from '~/src/api/decommission-service/controller.js'
+
+const decommissionService = {
+  plugin: {
+    name: 'decommission-service',
+    register: async (server) => {
+      await server.route([
+        {
+          method: 'DELETE',
```

Comment:
Do we do proper REST methods? 
Or do we need a fallback  POST? 
I see there is one PATCH in SSOP.

Reply:
Since this is a backend I'd guess it is fine as not browser based

Reply:
Seems fairly easy to change down the line if we need to 🤷‍♂️

Reply:
> Do we do proper REST methods?
FYI USBE uses most of the REST methods